logs/README

```diff
@@ -0,0 +1 @@
+THE LOGS FROM ACID AND IT'S FEATURES WILL BE STORED HERE
```

Comment:
Need to remake this, and have the logging code in ACID be ready for a filepath specification that does not exist in the system.
look here: https://stackoverflow.com/questions/20666764/python-logging-how-to-ensure-logfile-directory-is-created?rq=1